packages/coze-taro/global.d.ts

```
@@ -0,0 +1,20 @@
+  data?: Record<string, string>;
+}) => {
+  close(): void;
+  onMessage(
+    cb: (msg: { data: string; event: string; lastEventId: string }) => void,
+  ): void;
+  onOpen(cb: () => void): void;
+  onClose(cb: () => void): void;
+  onError(cb: (msg: { errMsg: string }) => void): void;
+};
+
+declare module globalThis {
```

Biome (1.9.4)

[error] Use the namespace keyword instead of the outdated module keyword.

The module keyword is deprecated to avoid any confusion with the ECMAScript modules which are often called modules.
Safe fix: Use namespace instead.


(lint/suspicious/useNamespaceKeyword)